pygen/pygen_src/riscv_directed_instr_lib.py

```diff
@@ -45,6 +45,45 @@ def post_randomize(self):
             self.instr_list[0].has_label = 1
 
 
+@vsc.randobj
+class riscv_mem_access_stream(riscv_directed_instr_stream):
+    def __init__(self):
+        super().__init__()
+        self.max_data_page_id = vsc.int32_t()
+        self.load_store_shared_memory = 0
+        self.data_page = {}
+
+    def pre_randomize(self):
+        if self.load_store_shared_memory:
+            self.data_page = cfg.amo_region
+        elif self.kernel_mode:
+            self.data_page = cfg.s_mem_region
+        else:
+            self.data_page = cfg.mem_region
+        self.max_data_page_id = len(self.data_page)
+
+    def add_rs1_init_la_instr(self, gpr, idx, base = 0):
+        la_instr = riscv_pseudo_instr()
+        la_instr.pseudo_instr_name = riscv_pseudo_instr_name_t.LA
+        la_instr.rd = gpr
+        if self.load_store_shared_memory:
+            la_instr.imm_str = "{}+{}".format(cfg.amo_region[idx]['name'], base)
+        elif self.kernel_mode:
+            la_instr.imm_str = "{}{}+{}".format(pkg_ins.hart_prefix(self.hart),
+                                                cfg.s_mem_region[idx]['name'], base)
+        else:
+            la_instr.imm_str = "{}{}+{}".format(pkg_ins.hart_prefix(self.hart),
+                                                cfg.mem_region[idx]['name'], base)
+        self.instr_list.insert(0, la_instr)
+
+    def add_mixed_instr(self, instr_cnt):
+        self.setup_allowed_instr(1, 1)
+        for i in range(instr_cnt):
+            instr = riscv_instr()
+            instr = self.randomize_instr(instr, [])
```

Comment:
Why do we need the second argument for the randomize_instr method?

Reply:
Hi @hodjat91 
Previously there was no default value in randomize_instr() for include_group. Later I decided to make the include_group positional in the function.
After the changes, I forgot to remove the [] from here.
Thanks for pointing this out.